rust/tool/src/esp.rs

```diff
@@ -67,11 +98,14 @@ pub struct EspGenerationPaths {
     pub kernel: PathBuf,
     pub initrd: PathBuf,
     pub lanzaboote_image: PathBuf,
+    pub efi_fallback: PathBuf,
+    pub systemd_boot: PathBuf,
 }
 
 impl EspGenerationPaths {
     pub fn new(esp_paths: &EspPaths, generation: &Generation) -> Result<Self> {
         let bootspec = &generation.spec.bootspec;
+        let system: System = System::from_str(&generation.spec.bootspec.system)?;
```

Comment:
Are we assuming that the `system` will change between generations? Otherwise this doesn't seem right. Maybe we should add a parameter to `lzbt` that tells it the system before reading any bootspecs.

Reply:
Otherwise, we could take this less correct approach of installing lanzaboote for the current system sure. I don't have a strong opinion, I find this implementation elegant because it does handle multi targets for free.